app.js

```diff
@@ -13,22 +13,31 @@ var config = {
 
 require('./index.js').connect(function (context) {
 
+    context.fs.readdir(context.path.join(__dirname, 'api', 'models'), (err, files) => {
+        //Load ALL modules.
+        //Don't get confused by weird logging order! console.log is async and will mix up the actual loading order.
+        files.forEach(file => {
+            context.component('models').module(file.replace('.js',''));
```

Comment:
Currently all "models" are loaded into context in Index.js .Should we do it again here ?
What difference does this make ?
What exactly is the fix?